From b308135e5c47c7f7eb3ef7fc0526f117a817f7f1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Wed, 16 Feb 2011 10:51:35 +0000
Subject: [PATCH] git-patch-id.txt: show that you can pipe in git-log
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the manual page for git-patch-id(1) to mention in the SYNOPSIS
that you can pipe in the git-log(1) output.

The manual page only mentioned that you could pipe in `< patch`, I had
to grep the test suite to find that you could do things like `git log
-p -1 | git patch-id`.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-patch-id.txt |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Documentation/git-patch-id.txt b/Documentation/git-patch-id.txt
index 4dae139..fec2cf3 100644
--- a/Documentation/git-patch-id.txt
+++ b/Documentation/git-patch-id.txt
@@ -9,6 +9,8 @@ SYNOPSIS
 --------
 'git patch-id' < <patch>
 
+'git log --pretty=format:%H -p | git patch-id'
+
 DESCRIPTION
 -----------
 A "patch ID" is nothing but a SHA1 of the diff associated with a patch, with
-- 
1.7.2.3

